### ISSUE TITLE
Finetuning SlowFast to custom dataset

### ISSUE THREAD

**Issue by tanveer-sayyed** [facebookresearch/SlowFast#518]
Hi,

The library is amazing! Thank you for your work.

I want to fine tune SlowFast[Kinetics-400] to my own custom dataset which has 2 labels.
All videos are approximately between 8-10 secs.
I have converted all videos to frames by keeping in mind num_frames and clip_duration so that transformation is as per SlowFast requirements. (But would like to get this confirmed ... !)

So can you please share the fine-tuning code/config file me?
(Before opening this issue I tried reaching out to a few persons who received the code/config from [here](https://github.com/facebookresearch/SlowFast/issues/18). But I did not receive any response, hence this issue.)

**Comment by tanveer-sayyed** [comment]
Do you want me to upload the file structure of the frames created?
Or anything?
Do let me know.

**Comment by dongxiaolong** [comment]
> Hi,
> 
> The library is amazing! Thank you for your work.
> 
> I want to fine tune SlowFast[Kinetics-400] to my own custom dataset which has 2 labels. All videos are approximately between 8-10 secs. I have converted all videos to frames by keeping in mind num_frames and clip_duration so that transformation is as per SlowFast requirements. (But would like to get this confirmed ... !)
> 
> So can you please share the fine-tuning code/config file me? (Before opening this issue I tried reaching out to a few persons who received the code/config from [here](https://github.com/facebookresearch/SlowFast/issues/18). But I did not receive any response, hence this issue.)

I meet the common problem. I want to know how to handle the problem  if  you solve it .

**Comment by fernanpatrickf** [comment]
Hello @tanveer-sayyed, have you resolved this or got any script for fine-tuning on custom dataset? Would appreciate if you could help me too. Thank you!

**Comment by tanveer-sayyed** [comment]
Hi @fernanpatrickf,
Please refer to this link first -> https://pytorch.org/hub/facebookresearch_pytorchvideo_slowfast/
There I got to know how the model runs. All you have to do is wrap a dataloader around it.
